### PR TITLE
FIX: Notifications pop up does not clear on devices without Google Se…

### LINKED ISSUE #1839: Notifications pop up does not clear on devices without Google Services (100k sats bounty)
The pop-up to enable notifications doesn't go away on my bluewallet. There are two options... ask me later, and turn on notifications

Turning on the notifications does nothing, because without google services, it can't turn on. The setting immediately returns to the off position. It will ask me the same question the next time I am sending funds.

Ask me later... just asks me again the next time I am sending funds also.

Please make a "No" button, that disables the query upon sending funds.

### PR DESCRIPTION
**Pull request by Overtorment**
…rvices (closes #1839)

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jbkjgh8nm6xcxx9vq0fa688arg

[download apk](http://filestorage.bluewallet.io:1488/1605118461.apk)

File: blue_modules/notifications.js [hunk 1]
```diff
@@ -118,6 +119,10 @@ function Notifications(props) {
     });
   };
 
+  Notifications.cleanUserOptOutFlag = async function () {
```
[the remark this answers is not in the document]
Comment (Overtorment) [reply blue_modules/notifications.js:122]: not sure. but in calling code we __do__ need to wail for operation to finish, so the code further down the line will see that theres no flag

[the remark this answers is not in the document]
Comment (marcosrdz) [reply blue_modules/notifications.js:122]: I think the return function will let it know its a Promise.

**Review by marcosrdz — APPROVED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**